.gitignore

```diff
@@ -56,3 +56,5 @@ rspec.xml
 
 # Ignore certificates directory
 config/certs
+
+dump.rdb
```

Comment:
what's this?

Reply:
@kreek - It is a file that repeatedly presents itself, during local dev, when using redis.  Perhaps this is exclusive to local dev w/o docker.  But yeah, I end up manually trashing it each time.